### PR TITLE
Локализация docs/1.getting-started/5.transitions.md (#24)

### LINKED ISSUE #24: Локализация docs/getting-started/transitions.md
REF: https://nuxt-ru.vercel.app/docs/getting-started/transitions

### PR DESCRIPTION
**Pull request by JonikUl**
### 🔗 Linked issue

resolves #24

### 📚 Description

Локализация [docs/1.getting-started/5.transitions.md](https://nuxt-ru.vercel.app/docs/getting-started/transitions)

### CODE REVIEW

**Comment by autofix-troubleshooter** [comment]
Hi! I'm the <a href="https://autofix.ci">autofix logo<strong>autofix.ci</strong></a> troubleshooter bot.

It looks like you correctly set up a CI job that uses the autofix.ci GitHub Action, but the [**autofix.ci GitHub App**](https://github.com/apps/autofix-ci/) has not been installed for this repository. This means that autofix.ci unfortunately does not have the permissions to fix this pull request. If you are the repository owner, please install the app and then restart the CI workflow! 😃

File: docs/1.getting-started/5.transitions.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-title: 'Transitions'
-description: Apply transitions between pages and layouts with Vue or native browser View Transitions.
+title: 'Переходы'
```
Comment (Hekikai) [inline docs/1.getting-started/5.transitions.md:2]: Давайте не переводить, как в доке по вью  
Transitions оставим

File: docs/1.getting-started/5.transitions.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ export default defineNuxtConfig({
 })
 ```
 
-To start adding transition between your pages and layouts, add the following CSS to your [`app.vue`](/docs/guide/directory-structure/app):
+Чтобы начать добавлять переходы между страницами и лейаутвми, добавьте следующий CSS в ваш [`app.vue`](/docs/guide/directory-structure/app):
```
Comment (Hekikai) [inline docs/1.getting-started/5.transitions.md:128]:
```suggestion
Чтобы начать добавлять переходы между страницами и лейаутами, добавьте следующий CSS в ваш [`app.vue`](/docs/guide/directory-structure/app):
```

File: docs/1.getting-started/5.transitions.md [hunk 3]
```diff
@@ -453,23 +453,22 @@ definePageMeta({
 ```
 
 ::alert{type="warning"}
-Overriding view transitions on a per-page basis will only have an effect if you have enabled the `experimental.viewTransition` option.
+Переопределение переходов представлений на уровне страницы будет иметь эффект только в том случае, если включена опция `experimental.viewTransition`.
 ::
 
-If you are also using Vue transitions like `pageTransition` and `layoutTransition` (see above) to achieve the same result as the new View Transitions API, then you may wish to _disable_ Vue transitions if the user's browser supports the newer, native web API. You can do this by creating `~/middleware/disable-vue-transitions.global.ts` with the following contents:
+Если вы также используете переходы Vue, такие как `pageTransition` и `layoutTransition` (см. выше), чтобы достичь того же результата, что и в новом View Transitions API, то вы можете захотеть _отключить_ переходы Vue, если браузер пользователя поддерживает более новый, нативный веб-API. Вы можете сделать это, создав `~/middleware/disable-vue-transitions.global.ts` со следующим содержимым:
```
Comment (Hekikai) [inline docs/1.getting-started/5.transitions.md:459]:
```suggestion
Если вы также используете переходы Vue, такие как `pageTransition` и `layoutTransition` (см. выше), чтобы достичь того же результата, что и в новом View Transitions API, то вы можете захотеть _отключить_ переходы Vue, если браузер пользователя поддерживает более новый, нативный Web API. Вы можете сделать это, создав `~/middleware/disable-vue-transitions.global.ts` со следующим содержимым:
```

File: docs/1.getting-started/5.transitions.md [hunk 4]
```diff
@@ -453,23 +453,22 @@ definePageMeta({
...
 })
 ```
 
-### Known issues
+### Известные проблемы
 
-- View transitions may not work as expected with nested pages/layouts/async components owing to this upstream Vue bug: <https://github.com/vuejs/core/issues/5513>. If you make use of this pattern, you may need to delay adopting this experimental feature or implement it yourself. Feedback is very welcome.
-
-- If you perform data fetching within your page setup functions, that you may wish to reconsider using this feature for the moment. (By design, View Transitions completely freeze DOM updates whilst they are taking place.) We're looking at restrict the View Transition to the final moments before `<Suspense>` resolves, but in the interim you may want to consider carefully whether to adopt this feature if this describes you.
+- Переходы между представлениями могут работать не так, как ожидается, с вложенными страницами/макетами/асинхронными компонентами из-за этой ошибки в восходящем потоке Vue: <https://github.com/vuejs/core/issues/5513>. Если вы используете этот шаблон, вам, возможно, придется отложить принятие этой экспериментальной функции или реализовать ее самостоятельно. Отзывы очень приветствуются.
```
Comment (Hekikai) [inline docs/1.getting-started/5.transitions.md:473]:
```suggestion
- Переходы между представлениями могут работать не так, как ожидается, с вложенными страницами/лейаутами/асинхронными компонентами из-за этой ошибки в известном баге Vue: <https://github.com/vuejs/core/issues/5513>. Если вы используете этот способ, вам, возможно, придется отложить принятие этой экспериментальной функции или реализовать ее самостоятельно. Отзывы очень приветствуются.
```

File: docs/1.getting-started/5.transitions.md [hunk 5]
```diff
@@ -453,23 +453,22 @@ definePageMeta({
...
+- Если вы выполняете выборку данных в функциях настройки страницы, возможно, вам стоит пересмотреть использование этой функции на данный момент. (По замыслу, View Transitions полностью останавливают обновления DOM, пока они происходят.) Мы рассматриваем возможность ограничить View Transition последними моментами перед разрешением `<Suspense>`, но пока вам стоит тщательно обдумать, стоит ли использовать эту функцию, если это относится к вам.
```
Comment (Hekikai) [inline docs/1.getting-started/5.transitions.md:474]:
```suggestion
- Если вы выполняете выборку данных в setup-функциях страницы, возможно, вам стоит пересмотреть использование этой функции на данный момент. (По замыслу, View Transitions полностью останавливают обновления DOM, пока они происходят.) Мы рассматриваем возможность ограничить View Transition до момента разрешения`<Suspense>`, но пока вам стоит тщательно обдумать, стоит ли использовать эту функцию, если это относится к вам.
```

**Comment by JonikUl** [comment]
@Hekikai пофиксил)

**Review by Ibochkarev — APPROVED** [review]

**Review by Hekikai — APPROVED** [review]

**Pull request merged**